cvat-ui/src/utils/opencv-wrapper/opencv-wrapper.ts

```diff
@@ -139,6 +139,20 @@ export class OpenCVWrapper {
             hist: () => new HistogramEqualizationImplementation(this.cv),
         };
     }
+
+    public get tracking(): Tracking {
+        if (!this.initialized) {
+            throw new Error('Need to initialize OpenCV first');
+        }
+        return {
+            trackerMIL: {
+                model: () => new TrackerMImplementation(this.cv),
+                name: 'TrackerMIL',
+                description: 'TrackerMIL',
+                type: 'opencv_tracker_mil',
```

Comment:
Where it is used?

Reply:
Name and description are used to render tracker list in side-bar. Type is used in two places:  in `INTERACT_WITH_CANVAS` reducer it is used to determine activeControl and in `repeatDrawShapeAsync` action it is used to provide correct interaction data: `shapeType: rectangle` and disable blocking algorithm button